code/modules/security/crimes.dm

```diff
@@ -0,0 +1,271 @@
+#define MAX_TIMER 10 HOURS //Permabrig.
+#define PRESET_SHORT 5 MINUTES
+#define PRESET_MEDIUM 10 MINUTES
+#define PRESET_LONG 15 MINUTES
+
+#define CRIME_MINOR "Minor"
+#define CRIME_MISDEMEANOUR "Misdemeanour"
+#define CRIME_MAJOR "Major"
+#define CRIME_CAPITAL "Capital"
+
+/datum/crime
```

Comment:
Please put a comment saying that these are overridden by config and shouldn't be used.